### PR TITLE
s3 task aws configuration

### PR DESCRIPTION
**Pull request by dknochen**

### CODE REVIEW

**Comment by dknochen** [comment]
@wcandillon can you please review? thanks

**Comment by wcandillon** [comment]
@dknochen by default the task looks up at `AWS_ACCESS_KEY_ID` `AWS_SECRET_ACCESS_KEY`. Having these variables configured in your system should be much less painful than setting up a secret file manually on your repo every time. What do you think?

I think we used to do that in sausalito

File: Gruntfile.js [hunk 1]
```diff
@@ -16,9 +16,24 @@ module.exports = function (grunt) {
...
+            /*  grunt-aws.json should look like this:
+                {
+                    "key": "***",
+                    "secret": "***",
+                    "region": "us-east-1",
+                }
+            */
+            return grunt.file.readJSON('config-aws.json');
+        }
+        grunt.log.warn('INFO: no config-aws.json file found. The "s3" task will not be available.');
```
Comment (wcandillon) [inline Gruntfile.js:30]: by default the task looks up `AWS_ACCESS_KEY_ID` `AWS_SECRET_ACCESS_KEY` so in that case the s3 task would be available.

Comment (dknochen) [inline Gruntfile.js:30]: @wcandillon I added an issue to document this (https://github.com/28msec/nolap-report-editor/issues/94)

**Pull request closed without merging**